test/components/attribute-picker/attribute-picker.test.js

```diff
@@ -145,5 +145,28 @@ describe('attribute-picker', () => {
 			expect(detail.limit).to.equal(5);
 			expect(component.attributeList.length).to.equal(5);
 		});
+
+		it('should fire the d2l-labs-attribute-picker-text-changed event when inputting text into the text input', async() => {
+			const component = await createComponent(attributeList, { limit: 5 });
+
+			const input = component.shadowRoot.querySelector('input');
+
+			await focusElem(input);
+
+			const listener1 = oneEvent(component, 'd2l-labs-attribute-picker-text-changed');
+			await sendKeysElem(input, 'type', 'a');
+			const { detail: detail1 } = await listener1;
+			expect(detail1.text).to.equal('a');
```

Comment:
Very nit, but to avoid having to separate out the `oneEvent` listener you should be able to exclude the `await` on `sendKeysElem`. Like this:

```suggestion
			sendKeysElem(input, 'type', 'a');
			const { detail: detail1 } = await oneEvent(component, 'd2l-labs-attribute-picker-text-changed');
			expect(detail1.text).to.equal('a');
```